redfish_utilities/inventory.py

```diff
@@ -0,0 +1,281 @@
+"""
+Sensors Module
```

Comment:
System Inventory Module?

Reply:
Whoops! I'll fix it when I'm in the office tomorrow